docs/en/observability/synthetics-analyze.asciidoc

```diff
@@ -0,0 +1,358 @@
+[[synthetics-analyze]]
+= Analyze data from synthetic monitors
+
+++++
+<titleabbrev>Analyze monitor data</titleabbrev>
+++++
+
+The Synthetics UI in {kib} both gives you a high-level overview of your service's
+availability and allows you to dig into details to diagnose what caused downtime.
+
+[discrete]
+[[synthetics-analyze-overview]]
+= Overview
+
+The Synthetics *Overview* tab provides you with a high-level view of all the services you are monitoring
+to help you quickly diagnose outages and other connectivity issues within your network.
+
+To access this page, go to *{observability}* -> *Synthetics* and make sure you're on the *Overview* tab.
+
+This overview includes a snapshot of the current status of all monitors, the number of errors that
+occurred over the last 6 hours, and the number of alerts over the last 12 hours.
+All monitors included your deployment will be listed below with information about the location,
+current status, and duration average.
+
+[NOTE]
+====
+When you use a single monitor configuration to create monitors in multiple locations, each location
+is listed as a separate monitor as they run as individual monitors and the status and duration average
+can vary by location.
+====
+
+[role="screenshot"]
+image::images/synthetics-monitor-page.png[Synthetics UI in {kib}]
+
+To get started with your analysis in the Overview tab, you can search for monitors or
+use the filter options including current status (up, down, or disabled),
+monitor type (for example, journey or HTTP), location, and more.
+
+Then click an individual monitor to see more details.
```

Comment:
Might be worth calling out that this will be through a flyout to distinguish that "go to a monitor" is a defined action, in the next section?